vfs/disk_usage_unix.go

```diff
@@ -2,16 +2,16 @@
 // of this source code is governed by a BSD-style license that can be found in
 // the LICENSE file.
 
-//go:build darwin || openbsd || dragonfly || freebsd
-// +build darwin openbsd dragonfly freebsd
+//go:build darwin || openbsd || dragonfly || freebsd || netbsd
+// +build darwin openbsd dragonfly freebsd netbsd
 
 package vfs
 
 import "golang.org/x/sys/unix"
 
 func (defaultFS) GetDiskUsage(path string) (DiskUsage, error) {
-	stat := unix.Statfs_t{}
-	if err := unix.Statfs(path, &stat); err != nil {
+	stat := unix.Statvfs_t{}
```

Comment:
Wont this break the other existing Unix builds? For example, I couldn't see `Statvfs_t` for Darwin or FreeBSD.

Suggest taking the approach that we took for Linux, and making a `disk_usage_netbsd.go` with the `netbsd` build tag, and then modifying the function we have here to call `Statvfs` with a `Statvfs_t` struct instead.

Reply:
Argh, great catch (and yay CI/CD pipeline!). I've tested only on NetBSD. Definitely that sounds safer and I will adjust it in a completely different `disk_usage_netbsd.go` so we are sure we don't break possible existing platforms!

Thanks!

Reply:
Yes, I can confirm it will break all other Unix builds except Solaris. I have reimplemented it as suggested.

Thank you Nick!

(Marking as resolved, both suggestions should be applied feel free to reopen it if there is anything left!)